src/guides/v2.3/release-notes/ReleaseNotes2.3.0Commerce.md

```diff
@@ -3682,6 +3668,8 @@ We've fixed hundreds of issues in the Magento 2.3.0 core code.
 
 *  Renaming of tables is not supported.
 
+**Known issue:** You cannot use the Magento Extension Manager to install extensions purchased from the Magento Marketplace. **Workaround**: Install extensions from the command line as described in [General CLI installation](https://devdocs.magento.com/extensions/install/).  See [Extension Manager shows no extensions in Magento Commerce 2.3.x](https://support.magento.com/hc/en-us/articles/360043980352).
```

Comment:
Global comment:

Do not use hard-coded URLs to link to devdocs. Replace `https://devdocs.magento.com` with `{{ site.baseurl }}`

Test locally on one file before replicating the change elsewhere.